lib/int_32.js

```diff
@@ -38,7 +38,7 @@ class Int32 {
    * @ignore
    */
   toExtendedJSON(options) {
-    if (options && (options.relaxed || options.legacy)) return this.value;
+    if (options && (options.relaxed || options.legacy)) return parseInt(this.value, 10);
```

Comment:
If we make a change in https://github.com/mongodb/js-bson/pull/359 to ensure that `Int32` is _always_ wrapping a `number`, then why do we ever need to `parseInt` again?

Reply:
I think this was clarified in meeting, as picking one of the solutions. Sorry for the confusion. Closing this PR.